apps/studio/components/interfaces/Database/Extensions/Extensions.tsx

```diff
@@ -73,49 +72,43 @@ const Extensions = () => {
       )}
 
       {isLoading ? (
-        <div className="my-8 w-full space-y-12">
-          <div className="space-y-4">
-            <ShimmeringLoader className="h-[28px] w-40" />
-
-            <div className="mb-4 grid grid-cols-1 gap-3 md:grid-cols-2 xl:grid-cols-3">
-              {Array.from({ length: 6 }).map((_, index) => (
-                <ExtensionCardSkeleton key={index} index={index} />
-              ))}
-            </div>
-          </div>
-        </div>
+        <GenericSkeletonLoader />
       ) : (
         <>
-          {extensions.length === 0 && (
+          {extensions.length === 0 ? (
             <NoSearchResults
```

Comment:
thinking the empty state could also show the headers

<img width="1464" height="202" alt="image" src="https://github.com/user-attachments/assets/daea79d2-8bad-4f3b-a3ba-66e3a859e0bd" />